### PR TITLE
[S] Documentation signpost added

### PR DESCRIPTION
**Pull request by e-kharion**
* added new documentation readme page which contains categorized links to all other documentation pages in this repository with brief description
* documentation section in repo readme file updated with a link to documentation readme

### CODE REVIEW

**Comment by e-kharion** [comment]
> @e-kharion please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @dotnet-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @dotnet-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @dotnet-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@dotnet-policy-service agree company="Microsoft"

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# .NET Performance Repository Documentation Readme
+
+The documentation in this repo is organized into the following sections:
+
+## Getting Started
+
+- [Prerequisites](prerequisites.md) - Information on what you need to get started.
+- [Perfreport walktrough](perfreport-walktrough.md) - This document describes Performance Report Advanced Features.
```
Comment (DrewScoggins) [inline docs/README.md:8]: nit: walktrough -> walkthrough this is also the case for the link itself

Perfreport -> Perf report

**Review by DrewScoggins — COMMENTED** [review]
Few spelling things, but otherwise LGTM.

File: docs/README.md (hunk 1 above)
Comment (e-kharion) [reply docs/README.md]: thanks, fixed

**Review by DrewScoggins — APPROVED** [review]
LGTM

**Review by LoopedBard3 — APPROVED** [review]
This looks good to me 👍.

**Pull request merged**